src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -61,8 +63,8 @@ public sealed partial class ComClassAttribute : System.Attribute
         public ComClassAttribute(string _ClassID, string _InterfaceID) { }
```

Comment:
>string [](start = 50, length = 6)

`string?`

Reply:
I can change these, but I figured that even though it doesn't throw for null, the intent was that you don't pass null in.  I should still change them?

Reply:
I'll just change 'em.  I'm betting on the fact that it'll affect literally 0 people :)